### PR TITLE
Merge administrator controls download permissions view into single xml

### PR DESCRIPTION
**Pull request by Akshatkamboj14**
## Explanation
solves #3411

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

| **BEFORE**  | **AFTER** |
| ------------- | ------------- |
| Screenshot_20220205_074706 | Screenshot_20220207_190240 |
| Screenshot_20220205_074800| Screenshot_20220207_190329 |
| Screenshot_20220205_080734 | Screenshot_20220207_223957 |
| Screenshot_20220205_080916 | Screenshot_20220207_224030 |

### CODE REVIEW

**Comment by Akshatkamboj14** [comment]
my oppia app is not working on the tablet even before applying changes, it doesn't show any error but the app is crashing only when I am running it on the tablet otherwise on the phone it's working properly.

**Comment by Akshatkamboj14** [comment]
This is the error....
Screenshot 2022-02-07 193828
Screenshot 2022-02-07 193908
@rishidyno @rt4914

**Comment by ayush0402** [comment]
@Akshatkamboj14 Hey, I just tried running this PR locally and though I wasn't getting any error during the build but the app was crashing on the first screen on Pixel C API 28 tablet. But Cleaning and Rebuilding the project through the BUILD menu on toolbar has fixed that crash for me. Did you try doing so?

**Comment by Akshatkamboj14** [comment]
@ayush0402 thank you for your help now it's working fine.

**Comment by Akshatkamboj14** [comment]
@ayush0402 @rishidyno can you please check this failed test?

File: app/src/main/res/layout/administrator_controls_download_permissions_view.xml [hunk 1]
```diff
@@ -141,7 +141,7 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginStart="40dp"
-        android:layout_marginEnd="4dp"
+        android:layout_marginEnd="@dimen/administrator_controls_download_permissions_view_lower_switch_compat_margin_end"
```
Comment (rishidyno) [inline app/src/main/res/layout/administrator_controls_download_permissions_view.xml]: @Akshatkamboj14  please explain `lower` in this dimension name? I suggest `administrator_controls_download_permissions_view_widget_switch_compat_margin_end` instead

File: app/src/main/res/values-sw600dp-land/dimens.xml [hunk 2]
```diff
@@ -420,4 +420,14 @@
   <dimen name="revision_card_fragment_layout_text_margin_top">32dp</dimen>
   <dimen name="revision_card_fragment_layout_button_margin_top">24dp</dimen>
 
+  <!--  AdministratorControlDownloadPermissionsView-->
```
Comment (rishidyno) [inline app/src/main/res/values-sw600dp-land/dimens.xml]: Please make sure spacing Is correct in comments.  e.g. `<!-- FAQ Single -->`
Ditto for others.

File: app/src/main/res/layout/administrator_controls_download_permissions_view.xml [hunk 3]
```diff
@@ -73,7 +73,7 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginStart="40dp"
-        android:layout_marginEnd="4dp"
+        android:layout_marginEnd="@dimen/administrator_controls_download_permissions_view_upper_switch_compat_margin_end"
```
Comment (rishidyno) [inline app/src/main/res/layout/administrator_controls_download_permissions_view.xml]: I suggest `administrator_controls_download_permissions_view_widget_switch_compat_margin_end` instead
Ditto for others.

File: app/src/main/res/layout/administrator_controls_download_permissions_view.xml [hunk 4]
```diff
@@ -18,9 +18,9 @@
       style="@style/CaptionViewStart"
       android:layout_width="match_parent"
       android:background="@color/mid_grey_30"
-      android:paddingStart="16dp"
+      android:paddingStart="@dimen/administrator_controls_download_permissions_view_upper_text_view_padding_start"
```
Comment (rishidyno) [inline app/src/main/res/layout/administrator_controls_download_permissions_view.xml]: @Akshatkamboj14 please explain upper in this dimension name ? 
I suggest `administrator_controls_download_permissions_view_download_permissions_text_view_padding_start` instead.

File: app/src/main/res/layout/administrator_controls_download_permissions_view.xml [hunk 5]
```diff
@@ -18,9 +18,9 @@
...
       android:paddingTop="28dp"
-      android:paddingEnd="16dp"
+      android:paddingEnd="@dimen/administrator_controls_download_permissions_view_upper_text_view_padding_end"
```
Comment (rishidyno) [inline app/src/main/res/layout/administrator_controls_download_permissions_view.xml]: Ditto. Remove `upper` and use `administrator_controls_download_permissions_view_download_permissions_text_view_padding_end` instead

File: app/src/main/res/layout/administrator_controls_download_permissions_view.xml [hunk 6]
```diff
@@ -38,9 +38,9 @@
       android:focusable="true"
       android:importantForAccessibility="yes"
       android:onClick="@{(view) -> viewModel.onTopicWifiUpdatePermissionChanged()}"
-      android:paddingStart="16dp"
+      android:paddingStart="@dimen/administrator_controls_download_permissions_view_upper_constraint_layout_padding_start"
```
Comment (rishidyno) [inline app/src/main/res/layout/administrator_controls_download_permissions_view.xml]: Ditto.

File: app/src/main/res/layout/administrator_controls_download_permissions_view.xml [hunk 7]
```diff
@@ -38,9 +38,9 @@
...
       android:paddingTop="20dp"
-      android:paddingEnd="16dp"
+      android:paddingEnd="@dimen/administrator_controls_download_permissions_view_upper_constraint_layout_padding_end"
```
Comment (rishidyno) [inline app/src/main/res/layout/administrator_controls_download_permissions_view.xml]: Ditto.

**Review by rishidyno — CHANGES_REQUESTED** [review]

**Comment by rishidyno** [comment]
@Akshatkamboj14 I suggest cleaning and rebuilding the project. You can find these options in `Build` menu

image

**Comment by Akshatkamboj14** [comment]
@rishidyno okay i will do it.

**Comment by Akshatkamboj14** [comment]
@rishidyno please take a look

**Pull request closed without merging**